mPowerSDK/Resources/JSONs/Base.lproj/APHConsentSection.json

```diff
@@ -117,9 +117,10 @@
      {
          "sectionType" : "custom",
          "sectionTitle": "We'll Test Your Understanding",
-         "sectionSummary": "It is important to know what this study is about and what is involved. Before you can join, we will ask you to take a short quiz to test your understanding.",
-        "sectionImage": "quiz_image",
-        "sectionHtmlContent" : "consent_2quiz_headsup"
+         "sectionSummary": "Before completing the enrollment it is important to understand what the study is about and what is involved.",
```

Comment:
This language change does not explain that participants will need to pass a quiz.  
Not accepted.